examples/connectivity/equinix-metal-to-fabric-connection/main.tf

```diff
@@ -8,7 +8,7 @@ provider "metal" {
 }
 
 data "metal_project" "this" {
-    name = var.metal_project_name
+  name = var.metal_project_name
```

Comment:
Could you either revert the formatting changes in the `examples` directory or open a separate PR with only the formatting changes?

Reply:
Will do the changes. Closing this PR in the favour smaller PRs per resource/data source